source/code-generation.txt

```diff
@@ -0,0 +1,56 @@
+.. _rm-code-generation:
+
+===============
+Code Generation
+===============
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Relational Migrator can assist you in converting both your SQL and 
+application code to MongoDB. By utilizing Relational Migrator's code 
+generation features, you can streamline the migration process and 
+save time and effort.
```

Comment:
[suggestion]

I'm not sure if this wording is a requirement from product, but I might consider removing "and save time and effort" to make this sound a little bit less like a marketing blurb.